Shared/Samples/Simplify geometry/README.metadata.json

```
@@ -0,0 +1,26 @@
+{
+    "category": "Layers",
```

```suggestion
    "category": "Edit and Manage Data",
```

I have been doing some category work and noticed that `Edit and Manage Data` might be more appropriate for this sample based on the description in `common-samples/blob/main/designs/categories.md`. I didn't review the rest of the PR.